src/components/Login/LoginContainer.tsx

```diff
@@ -140,18 +143,12 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
       lastResendWarningMessageRef.current = response.warning ?? "";
       return true;
     } catch (e) {
-      if (e instanceof LoginLockedError) {
-        showAlert({
-          ...disabledAccessAlertProps,
-          description: e.message,
-          onOk: () => resetStage()
-        });
+      if (e instanceof AuthError) {
+        showAlert({ ...e.alertProps, onOk: () => resetStage() });
+      } else if (e instanceof LoginLockedError) {
+        showAlert({ ...e.alertProps, onOk: () => resetStage() });
       } else if (e instanceof LoginError) {
-        showAlert({
-          ...invalidInputAlertProps,
-          description: e.message,
-          onOk: () => resetStage()
-        });
+        showAlert(e.alertProps);
```

Comment:
might have to add `resetStage` here too!

Reply:
it was stuck after spoofing the endpoint but providing a valid key.
and if we do add `resetStage` here, then we can combine all 3 error instances in this as one condition haha

Reply:
yeap changed haha